zipkin-collector/kafka/pom.xml

```diff
@@ -29,6 +29,7 @@
   <properties>
     <main.basedir>${project.basedir}/../..</main.basedir>
     <!-- when changing this, a change to the version of kafka-junit is likely needed, too -->
+    <!-- TODO: update to 2.6.0 once https://github.com/charithe/kafka-junit/issues/54 is complete -->
```

Comment:
Thanks for the reminder. I will check if testcontainers + zipkin kafka image can be used here to remove this dependency; similar to https://github.com/openzipkin-contrib/zipkin-storage-kafka/pull/70z

Reply:
yeah we can use our own "last image" so that the version bumping matches on things like our mysql tests.